pkg/cmd/upload-binaries/main.go

```diff
@@ -59,7 +60,13 @@ var libsRe = func() *regexp.Regexp {
 
 var osVersionRe = regexp.MustCompile(`\d+(\.\d+)*-`)
 
+var releaseMode bool
+
 func main() {
+	flag.BoolVar(&releaseMode, "release", false,
```

Comment:
we've historically declared flags in the `var` stanza. see: https://github.com/cockroachdb/examples-go/blob/master/block_writer/main.go for an example